### PR TITLE
Queue upgrade

### PR DESCRIPTION
**Pull request by tank157**
I don't know that this work was strictly necessary for the missing electorates fixes, but it was work I'd done to simplify/rule out possibilities, so I'd like it to go as well.

[Pivotal Card](https://www.pivotaltracker.com/story/show/122617581)

Assigning to @cap10morgan

### CODE REVIEW

File: src/electorate_works/channels.clj [hunk 1]
```diff
@@ -7,9 +7,9 @@
 (defonce delete-events             (async/chan 1000))
 
 ;; incoming
-(defonce user-create-events        (async/chan))
-(defonce user-update-events        (async/chan))
-(defonce election-create-events    (async/chan))
+(defonce user-create-events        (async/chan 1000))
+(defonce user-update-events        (async/chan 1000))
+(defonce election-create-events    (async/chan 1000))
```
Comment (cap10morgan) [inline src/electorate_works/channels.clj]: Do we need buffers on these? That's pretty different from what we normally do (we usually just put buffers on outgoing channels) and might mess up kehaar back-pressure, for example. The events should be able to queue up in Rabbit if this isn't handling them quickly enough.

But have you seen evidence that something like this is necessary?

File: src/electorate_works/queue.clj [hunk 2]
```diff
@@ -10,162 +10,164 @@
...
+            "user-works.user.search"
+            (config [:rabbitmq :queues "user-works.user.search"])
+            10000
+            channels/user-search)
+           (wire-up/external-service
+            connection
+            ""
+            "user-works.user.read"
+            (config [:rabbitmq :queues "user-works.user.read"])
+            10000
```
Comment (cap10morgan) [inline src/electorate_works/queue.clj]: Can we make all these timeouts 30s too?

File: src/electorate_works/channels.clj (hunk 1 above)
Comment (tank157) [inline src/electorate_works/channels.clj]: Nope, not sure why I added the buffers here, I'll remove them.

**Comment by cap10morgan** [comment]
LGTM

**Pull request merged**